grammar.y

```diff
@@ -972,7 +986,35 @@ create_table_stmt:
     if len($5) > MaxAllowedColumns {
       yylex.(*Lexer).AddError(&ErrTooManyColumns{ColumnCount: len($5), MaxAllowed: MaxAllowedColumns})
     }
-    $$ = &CreateTable{Table: $3, Columns: $5, Constraints: $6}
+
+    // We have to replace a primary key table constraint with an equivalent column constraint primary key,
```

Comment:
[L998-1015](https://github.com/tablelandnetwork/go-sqlparser/pull/29/files#diff-a6ec253416e8e9e6846f73f5c7677d6ff6fa85f5c37317f84e608fc50ff8238eR998-R1015) is the replace logic of a table constraint for a column constraint. LMK if that makes sense.